addons/mail/static/src/discuss/call/common/call_action_list.xml

```diff
@@ -9,7 +9,7 @@
                         <CallActionButton action="action" isSmall="isSmall" isActive="action.isActive"/>
                     </t>
                     <Dropdown position="'top-end'" menuClass="'d-flex flex-column py-0'">
-                        <button class="btn smaller d-flex m-1 border-0 rounded-circle shadow-none opacity-100 opacity-75-hover align-items-center p-0" t-att-title="MORE">
+                        <button class="btn smaller d-flex m-1 border-0 rounded-circle opacity-100 opacity-75-hover align-items-center p-0" t-att-title="MORE" t-ref="more" t-on-mouseenter="onEnterMore" t-on-mouseleave="onLeaveMore" t-on-click="onClickMore" t-att-class="{ 'o-discuss-CallActionList-pulse': props.thread.fullscreenIndicator === INDICATOR.ACTIVE, 'shadow-none': props.thread.fullscreenIndicator !== INDICATOR.ACTIVE }">
```

Comment:
Would rename `onEnterMore` and `onLeaveMore` to respectively `onMouseenterMore` and `onMouseleaveMore`. This is to follow idiomatic naming `onEventnamePartname`, so the name of the handler gives better grasp of flow.

The `t-att-class` can be made on multi-line, to breath a bit more on such a long LOC.